protocol/directory_test.go

```diff
@@ -80,6 +80,13 @@ func TestRegisterExistedUserWithTB(t *testing.T) {
 }
 
 func TestRegisterWithoutTB(t *testing.T) {
+	// workaround for #110
```

Comment:
Sorry, I didn't really understand your question. These tests should panic and rot right after directory initialization. Should we remove these tests?